CONTRIBUTING.md

```diff
@@ -30,6 +30,13 @@ Contributors can be nominated as Collaborators by Steering Committee members. On
 
 Nominations for Collaborator status happen through the typical Steering Committee decision making process. That is, to nominate one or more Collaborators, Steering Committee then moves to interview the nominees. The nominee is approved or rejected following the same consensus seeking process for new team members.
 
+#### Code Owners
+Collaborators can become a Code Owner for a specific package(s). They have the ability to approve a pull request that has made changes in those package(s) and at least one Code Owner will be required for any package.
```

Comment:
Is this rule implicit? Or is there a mechanism on GitHub to enforce this policy?

Reply:
The CODEOWNERS file in .github/ directory should enforce this by the rules set there once I can assure it's wired up correctly.